conf/modules.config

```diff
@@ -0,0 +1,250 @@
+/*
+========================================================================================
+    Config file for defining DSL2 per module options
```

Comment:
we should only have config for the tools we actually have here

Reply:
changed